web/src/app/administration/components/workbasket-details/workbasket-details.component.ts

```diff
@@ -108,12 +97,6 @@ export class WorkbasketDetailsComponent implements OnInit, OnDestroy {
     if (!workbasketIdSelected && this.action === ACTION.CREATE) {
       // CREATE
       this.workbasket = {};
-      this.domainService
-        .getSelectedDomain()
-        .pipe(takeUntil(this.destroy$))
-        .subscribe((domain) => {
-          this.workbasket.domain = domain;
-        });
```

Comment:
Why do we not need this anymore?

Reply:
I'm gonna restore this part, will need to discuss together if we should remove it